### ISSUE TITLE
Internationalization issue

### ISSUE THREAD

**Issue by oliynykroman** [VirtusLab-Open-Source/strapi-plugin-navigation#390]
Hi,
I have two languages on the site (EN, DK). When I added new language in internationalization (so i have EN - default, DK and NO) DK version in navigation plugin just deleted and replaced with NO (this happen after admin reload, before admin reload i see EN, DK versions). When i trying to add new navigation i see only EN version. 
![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/29798653/51585e84-fbf9-49d3-af7e-ce1863e6bd74)

After some time site stop working with error:  
![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/29798653/78beabb7-f9ab-460b-b335-c8ed44c76ce1)

_**error: insert into `navigations` (`created_at`, `locale_code`, `name`, `slug`, `updated_at`, `visible`) values ('2023-10-20 05:41:29.971', 'da-DK', 'Header', 'header-da-DK', '2023-10-20 05:41:29.971', true) - ER_DUP_ENTRY: Duplicate entry 'header-da-DK' for key 'navigations_slug_unique'.**_ 

Only remove localization manually in DB resolve problem. 

**Configuration:**
```
    "@strapi/plugin-i18n": "4.14.5",
    "@strapi/plugin-seo": "^1.9.5",
    "@strapi/plugin-users-permissions": "4.14.5",
    "@strapi/strapi": "4.14.5",
    "mysql": "2.18.1",
    "strapi-plugin-navigation": "^2.2.16",
    "strapi-plugin-populate-deep": "^3.0.0"
```

**Comment by damian-balas** [comment]
I have the same issue. Adding a language is causing issues. Same goes for deleting them.

**Comment by CodeVoyager** [comment]
Hi @oliynykroman and @damian-balas,

Thank you for your posting. We will investigate and keep you posted.

**Comment by CodeVoyager** [comment]
Hi @oliynykroman and @damian-balas,

PR with fix for this issue is in review. I reckon new version of the plugin will be available in about 2 weeks.

**Comment by cyp3rius** [comment]
Fixed in `v2.3.1`
